integration/setup/setup.go

```diff
@@ -105,7 +103,7 @@ func installResources(ctx context.Context, config Config) error {
 			return microerror.Mask(err)
 		}
 
-		err = config.K8sClients.CRDClient().EnsureCreated(ctx, chartCRD, backoff.NewMaxRetries(7, 1*time.Second))
+		err = config.K8sClients.CtrlClient().Create(ctx, chartCRD)
```

Comment:
I assume we can't configure the backoff here?

Reply:
Yes the CRD package in k8sclient had retry logic which was configured here. It was removed in k8sclient v6.